### PR TITLE
#786: Exports are now sorted correctly. Added tests for the sorting.

### PR DESCRIPTION
**Pull request by He3lixxx**

### CODE REVIEW

File: evap/results/exporters.py [hunk 1]
```diff
@@ -107,6 +107,7 @@ def export(self, response, course_types_list, include_not_enough_answers=False,
                     used_questionnaires.add(questionnaire)
                 courses_with_results.append((course, results))
 
+            courses_with_results.sort(key=lambda cr: cr[0].name)
             courses_with_results.sort(key=lambda cr: cr[0].type)
```
Comment (karyon) [inline evap/results/exporters.py]: found this: http://stackoverflow.com/questions/4233476/sort-a-list-by-multiple-attributes
```
courses_with_results.sort(key=lambda cr: (cr[0].type, cr[0].name))
```

File: evap/staff/tests/test_views.py [hunk 2]
```diff
@@ -84,9 +84,40 @@ def test_deletes_users(self):
...
+        page = self.app.get(self.url, user='staff').body.decode("utf-8")
+        position_course1 = page.find("Course 1")
+        position_course2 = page.find("Course 2")
+        self.assertLess(position_course1, position_course2)
+
+
 class TestSemesterExportView(ViewTest):
     url = '/staff/semester/1/export'
     test_users = ['staff']
+    extra_environ = {'HTTP_ACCEPT_LANGUAGE': 'en'}
```
Comment (karyon) [inline evap/staff/tests/test_views.py]: is this still necessary here (other test as well)? if not, i'd prefer to just set this inside the test

Comment (He3lixxx) [inline evap/staff/tests/test_views.py]: I moved it to the app.get() and form.submit() calls.

Comment (karyon) [inline evap/staff/tests/test_views.py]: that's even better.

**Pull request merged**